src/pkg/services/m365/api/channels_pager.go

```diff
@@ -144,6 +144,25 @@ func (c Channels) NewChannelMessageDeltaPager(
 	}
 }
 
+// this is the message content for system chatMessage entities with type
+// unknownFutureValue.
+const channelMessageSystemMessageContent = "<systemEventMessage/>"
+
+func FilterOutSystemMessages(cm models.ChatMessageable) bool {
```

Comment:
I'm not aware of the the reasoning behind this, but I feel like we should at least store it. I the the filter should be at the layer where we read it for export, details or other operation.

Reply:
Per sync conversation we've agreed to not store system event messages.  We can discuss further in cycle review if wanted.